pyfiglet/test.py

```diff
@@ -48,7 +51,13 @@ def outputUsingFigletorToilet(self, text, font, fontpath):
             raise Exception('Missing font file: '+fontpath)
 
         p = Popen(cmd, bufsize=4096, stdout=PIPE)
-        outputFiglet = p.communicate()[0].decode('utf8')
+        try:
+            outputFiglet = p.communicate()[0].decode('utf8')
+        except UnicodeDecodeError as e:
+            print("Unicode Error handling font {}".format(font))
+            outputFiglet = ''
+        except e:
+            raise e
```

Comment:
Probably. It if was anything other than a UnicodeDecodeError, I didn't want it to go by un-noticed.